doc/source/getting_started/overview.rst

```diff
@@ -72,7 +72,7 @@ Some other notes
    extensively tweaked in `Cython <https://cython.org>`__ code. However, as with
    anything else generalization usually sacrifices performance. So if you focus
    on one feature for your application you may be able to create a faster
-   specialized tool.
+  -specialized tool.
```

Comment:
I think this was fine as it was before too, and the hyphen seems out-of-line with the others. Did you try building this page of the docs locally?

Reply:
Yeah, I think that is causing a problem in the build. I'll revert it back.